src/app/qgsabout.cpp

```diff
@@ -271,6 +274,12 @@ void QgsAbout::setPluginInfo()
   txtProviders->setText( myString );
 }
 
+void QgsAbout::btnCopyToClipboard_clicked()
+{
+  const QString versionString = QgisApp::instance()->getVersionString();
+  QGuiApplication::clipboard()->setText( versionString );
+}
```

Comment:
Instead of the back and forth between generating the string in QgisApp, setting it for the dialog via the call QgsAbout::setVersion from app, and then here calling BACK to QgisApp to get the string again, I'd suggest we instead store the raw string in a QString member variable in QgsAbout, and set its value directly in QgsAbout::setVersion

That'd avoid a bit of spaghetti logic :+1:

Reply:
Sounds good to me, done  👍